axum/static-next-server/README.md

```diff
@@ -0,0 +1,3 @@
+# Serving a NextJS project with Axum
+
+This example shows how to serve a NextJS project using [axum](https://github.com/tokio-rs/axum) and shuttle.
```

Comment:
I would say this is essentially the same as our static files axum example, and not really an example of serving a nextjs project using axum. I know this is supposed to serve as a template for the node cli (and thus the extra bits in the other static files examples shouldn't be included), but it may be confusing for users seeing this in the examples repo. Maybe we can leave instructions for how to extract a next.js app to the static folder, or just link to Josh's article?